Gemfile.lock

```diff
@@ -121,36 +122,35 @@ GEM
       activesupport (>= 3.0.0)
       uniform_notifier (~> 1.11)
     byebug (11.1.3)
-    capistrano (3.14.0)
+    capistrano (3.14.1)
       airbrussh (>= 1.0.0)
       i18n
       rake (>= 10.0.0)
       sshkit (>= 1.9.0)
-    capistrano-bundler (1.6.0)
+    capistrano-bundler (2.0.1)
       capistrano (~> 3.1)
-    capistrano-rails (1.5.0)
+    capistrano-rails (1.6.1)
       capistrano (~> 3.1)
-      capistrano-bundler (~> 1.1)
+      capistrano-bundler (>= 1.1, < 3)
     capistrano-rails-console (2.3.0)
       capistrano (>= 3.5.0, < 4.0.0)
       sshkit-interactive (~> 0.3.0)
     case_transform (0.2)
       activesupport
-    code_analyzer (0.5.1)
+    code_analyzer (0.5.2)
```

Comment:
### [code_analyzer](https://github.com/flyerhzm/code_analyzer)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.5.1 → 0.5.2

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **14** commits. See the full changes on [the compare page](https://github.com/flyerhzm/code_analyzer/compare/v0.5.1...v0.5.2).

These are the first 10 commits:
- (d5e0221) [Auto corrected by following Lint Ruby MagicComment](https://github.com/flyerhzm/code_analyzer/commit/d5e02213ef285e7a0e86f2388f87d6f25ad193bc)
- (b7ea213) [Auto corrected by following Format Ruby Code](https://github.com/flyerhzm/code_analyzer/commit/b7ea2132dc9299ae3e418e13543309a321b981e6)
- (e7e93d1) [Merge pull request #10 from flyerhzm/awesomecode-lint-ruby-m…](https://github.com/flyerhzm/code_analyzer/commit/e7e93d139ec11c1ac3447e0430b770a7d5be0b64)
- (7d8eae2) [Auto corrected by following Lint Ruby EmptyLine](https://github.com/flyerhzm/code_analyzer/commit/7d8eae23ef79a0f255eedde448dd9ea23aadee08)
- (6de6c9b) [Merge pull request #12 from flyerhzm/awesomecode-lint-ruby-e…](https://github.com/flyerhzm/code_analyzer/commit/6de6c9bcbdc9c8f2b05d0619d5506911a6244f5a)
- (c59487d) [Fix `Sexp#line_number` on `:unary`](https://github.com/flyerhzm/code_analyzer/commit/c59487df59e9133d73046217f2785cd830761430)
- (331749f) [Merge pull request #13 from ybiquitous/fix-line-number-on-un…](https://github.com/flyerhzm/code_analyzer/commit/331749f94f4ff0f6b1b4cd976363057540fd826a)
- (825fc08) [Fix `Sexp#line_number` on various types](https://github.com/flyerhzm/code_analyzer/commit/825fc0836b42bf70e8bf32b967cc810ff1e919ba)
- (4309d92) [Update Ruby versions on Travis CI (updated: 2.6.6, added: 2.…](https://github.com/flyerhzm/code_analyzer/commit/4309d92ea070898c5e5083d18d71ec7e5e8ebfa1)
- (e31ce43) [Merge pull request #30 from ybiquitous/fix-sexp-line-number](https://github.com/flyerhzm/code_analyzer/commit/e31ce438d3858df055bf85334003f29e1bd1673d)


</details>